### DISCUSSION TITLE
Pixel data is lost when using CSTORE

### DISCUSSION THREAD

**Discussion by jtournas** [fo-dicom/fo-dicom#1656] [Q&A]
We’ve got a working c-store client however we’re noting off behavior. 

If you take a jpeg2000 image and send it to a device which doesn’t support jpeg2000 the resulting transfer is done in little endian implicit.  This is expected so far and little endian implicit was offered and accepted on the association. 

The issue is that once the image is transfer the pixel data has been stripped.  I’ve been able to reproduce this with two separate SCP 

Why is the pixel data not getting converted to native during the on the fly transcoding?

**Comment by kalinins** [comment]
[fodicom_test.zip](https://github.com/fo-dicom/fo-dicom/files/12715846/fodicom_test.zip)

Attached to this comment is a test project with a sample j2k image. Sending this image using fo-dicom cstore somehow strips pixel data.

**Comment by kalinins** [comment]
Apparently using fo-dicom.Codecs fixes this issue.
